pages/integrations/index.tsx

```diff
@@ -413,6 +413,14 @@ export async function getServerSideProps(context) {
     },
   });
 
+  if (!user.completedOnboarding) {
```

Comment:
Like calendly, before a "new user" has completed their onboarding all pages are "blocked" and redirected by to the onboarding. 

Not shown in code, but in order to not show this onboarding flow for existing users if they're account was created before ONBOARDING_INTRODUCED_AT a short cut to get around the onboarding is implemented. 





@emrysal ... another middleware in getServerSideProps lets explore some neat solutions to this.